notecard/notecard.go

```diff
@@ -164,26 +173,18 @@ func OpenSerial(port string, portConfig int) (context Context, err error) {
 	context.PortEnumFn = serialPortEnum
 	context.PortDefaultsFn = serialDefault
 	context.CloseFn = cardCloseSerial
+	context.ReopenFn = cardReopenSerial
 	context.ResetFn = cardResetSerial
 	context.TransactionFn = cardTransactionSerial
+
+	// Record serial configuration
 	context.isSerial = true
+	context.serialConfig.Name = port
+	context.serialConfig.Baud = portConfig
+	context.serialConfig.ReadTimeout = time.Millisecond * 500
 
 	// Open the serial port
-	///*	tarm
-	c := &serial.Config{}
-	c.Name = port
-	c.Baud = portConfig
-	c.ReadTimeout = time.Millisecond * 500
-	context.openSerialPort, err = serial.OpenPort(c)
-	//*/
-	/* jacobsa
-	   c := serial.OpenOptions{}
-	   c.PortName = port
-	   c.BaudRate = portConfig
-	   c.MinimumReadSize = 0
-	   c.InterCharacterTimeout = 5000
-	   context.openSerialPort, err = serial.Open(c)
-	*/
+	err = cardReopenSerial(&context)
```

Comment:
Here's where we stash serial context. Once we do that, the rest of the Open() operation is the same as Reopen().